src/mdoc/models/pex-limit-disclosure.ts

```diff
@@ -163,12 +166,14 @@ export const limitDisclosureToInputDescriptor = (
     const { namespace, digest } = result
     const entry = deviceNamespaces.get(namespace)
     if (!entry) {
+      issuerNamespaces.set(namespace, [digest])
```

Comment:
SAme here, device namespaces logic needs to be removed, and we only keep issuer namespaces.

Baiscally just copy the 0.5 implementation